lab-geoff/route/book-route.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+let storage = require('./../lib/storage.js');
+let response = require('./../lib/response.js');
+let Book = require('../model/book.js');
+
+module.exports = function(router) {
+  router.get('/books', function(req, res) {
+    if (req.url.query.id) {
+      storage.fetchItem('books', req.url.query.id)
+        .then(book => {
+          response.sendJSON(res, 200, book);
```

Comment:
This is not going to work. `response` is not defined, given that your parameters are `req` and `res`.